satellite/marketing/offers_test.go

```diff
@@ -59,12 +59,14 @@ func TestOffer_Database(t *testing.T) {
 			require.Equal(t, new, c)
 
 			update := &marketing.UpdateOffer{
-				ID:          new.ID,
-				Status:      marketing.Done,
-				NumRedeemed: new.NumRedeemed,
-				ExpiresAt:   time.Now(),
+				ID:        new.ID,
+				Status:    marketing.Done,
+				ExpiresAt: time.Now(),
 			}
-			err = planet.Satellites[0].DB.Marketing().Offers().Update(ctx, update)
+			err = planet.Satellites[0].DB.Marketing().Offers().Redeem(ctx, update.ID)
+			require.NoError(t, err)
+
+			err = planet.Satellites[0].DB.Marketing().Offers().Finish(ctx, update.ID)
```

Comment:
Are you checking that the it is doing the correct thing?

Reply:
Good point! I will add the check

Reply:
I think this is a good candidate for table driven tests. Not necessary but would be a good learning exercise.

https://github.com/golang/go/wiki/TableDrivenTests